internal/resources/ekscluster/data_source.go

```diff
@@ -99,7 +99,7 @@ func dataSourceTMCEKSClusterRead(ctx context.Context, d *schema.ResourceData, m
 	default:
 		timeoutDuration, parseErr := time.ParseDuration(timeoutValueData)
 		if parseErr != nil {
-			log.Printf("[INFO] unable to prase the duration value for the key %s. Defaulting to 5 minutes(5m)"+
+			log.Printf("[INFO] unable to prase the duration value for the key %s. Defaulting to 30 minutes(30m)"+
```

Comment:
nit:Fix typo : prase -> parse

Also in the log message instead of hardcoding `30 minutes(30m)` lets refer to the `const defaultTimeout`, this will avoid such inconsistencies  if this value is changed in future

Reply:
@ramya-bangera Please review updates to address your concerns.